editor/src/messages/portfolio/portfolio_message_handler.rs

```diff
@@ -110,6 +111,10 @@ impl MessageHandler<PortfolioMessage, (&InputPreprocessorMessageHandler, &Prefer
 					// Clear properties panel and layer tree
 					responses.push_back(PropertiesPanelMessage::ClearSelection.into());
 					responses.push_back(DocumentMessage::ClearLayerTree.into());
+
+					// Clear existing hints
```

Comment:
Can I have you please remove this comment and integrate it into the comment above, and also remove that blank line? Since they are all related in purpose. Maybe that earlier comment can just be more generic like `// Clear layouts that assume the existence of a document"